src/redux/incidents/sagas.js

```diff
@@ -136,6 +136,7 @@ export function* getIncidentsImpl() {
     incidents = yield call(pdParallelFetch, 'incidents', baseParams, null, {
       priority: 5,
       skipSort: true,
+      maxRecords: 10000,
```

Comment:
@martindstone - Does this actually get used?

Looks like it's just looking at incidents error to detect too many incidents
```
 if (!fetchingIncidents && incidents.length === 0 && incidentsError?.startsWith('Too many records')) {
 ```